modules/home-environment.nix

```
@@ -552,6 +552,9 @@ in
 
             cp ${activationScript} $out/activate
 
+            mkdir $out/bin
+            ln -s $out/activate $out/bin/home-manager-generation
```

```suggestion
            ln -s $out/activate $out/bin/activate
```

To have a similar API to NixOS.

---

When I tested it, nix run only tries to run `$out/bin/<package-name>` (that's why `nix run nixpkgs#neovim` doesn't work).
So if the symlink is moved to a different location you can't run it automatically anymore.

---

see NixOS/nix#4498

---

Right, I forgot about this peculiarity of `nix run`. It's good to know `meta.mainProgram` exists now.
I usually always do `nix shell nixpkgs#something -c some-program`